tree/dataframe/inc/ROOT/RDF/ActionHelpers.hxx

```diff
@@ -424,6 +424,17 @@ public:
 // 3. The column is an RVec, the collection is not a vector
 // 4. The column is an RVec, the collection is a vector
 
+template <typename T, typename COLL>
+void FillColl(T& v, COLL c) {
```

Comment:
passing `COLL` by value you would be constructing a `shared_ptr` at every entry. Probably better to take a `COLL&` and call `FillColl(v, *fColls[slot])`

Reply:
Thanks for pointing this out! Fixed.